app/menu.js

```diff
@@ -244,6 +244,18 @@ module.exports = ({createWindow, updatePlugins}) => {
             focusedWindow.rpc.emit('decrease fontSize req');
           }
         }
+      },
+      {
+        type: 'separator'
+      },
+      {
+        label: 'Maximize / Restore Pane',
```

Comment:
This text should depend on the current state, but I'm not sure how to handle it here.